tracim/tracim/tests/models/test_content.py

```diff
@@ -163,9 +172,42 @@ def test_create(self, key='1'):
 
         return created_content
 
+    def _get_user(self):
+        email = 'admin@admin.admin'
+        user_query = DBSession.query(User)
+        user_filter = user_query.filter(User.email == email)
+        user = user_filter.one()
+        return user
+
     def _create_content(self, *args, **kwargs):
         content = Content(*args, **kwargs)
         DBSession.add(content)
         DBSession.flush()
+        return content
 
+    def _create_content_from_nothing(self):
+        user_admin = self._get_user()
+        workspace = Workspace(label="TEST_WORKSPACE_1")
+        content = self._create_content(
+            owner=user_admin,
+            workspace=workspace,
+            type=ContentType.File,
+            label='TEST_CONTENT_1',
+            description='TEST_CONTENT_DESCRIPTION_1',
+            revision_type=ActionDescription.CREATION,
+        )
         return content
+
+    def test_unit__content_depot_file(self):
+        """ Depot file access thought content property methods. """
+        content = self._create_content_from_nothing()
+        # tests uninitialized depot file
+        eq_(content.depot_file, None)
+        # initializes depot file
+        content.depot_file = b'test'
```

Comment:
I did this comment before our discussion about UploadedFile field behavior. Forget the current comment :)